app/address/address-lookup.js

```diff
@@ -1,19 +1,18 @@
 const config = require('config');
-const extractAddress = require('./extract-address');
+// const extractAddress = require('./extract-address');
 const fetch = require('node-fetch');
 const HttpsProxyAgent = require('https-proxy-agent');
 const { Logger } = require('@hmcts/nodejs-logging');
 
 const logger = Logger.getLogger('addressLookup');
 
 
-function addressLookup(postcode) {
+function addressLookup() {
 
   return fetch(
-    config.address_lookup.url.replace('${postcode}', postcode), getHttpConfig())
+    config.address_lookup.url.replace('${key}', config.get('secrets.ccd.postcode-info-address-lookup-token')), getHttpConfig())
```

Comment:
We do it here as well as in the `ccd-case-management-web`?

Reply:
ccd-case-management-web will have to postcode lookup url to gateway configuration address and that will get passed postcode lookup component and the component will call the gateway URL with the postcode attribute and gateway will fill the api key value.

Reply:
Ok now I see it is update in man web 👍